packages/bitcoin/src/psbt/psbt-outputs.ts

```diff
@@ -36,9 +35,11 @@ export function getParsedOutputs({
         // logger.error('Output has no script');
         return;
       }
-      const outputAddress = createBitcoinAddress(
-        getAddressFromOutScript(output.script, bitcoinNetwork)
-      );
+      const outputAddress = getAddressFromOutScript(output.script, bitcoinNetwork);
+      if (outputAddress === null) {
+        throw new Error('PSBT output has unsupported bitcoin address');
```

Comment:
@edgarkhanzadian I think throwing this error here is going to break PSBTs for existing users. There was an edge case with 'unknown' where we needed to allow it. It also needs to be addressed in the extension bc we look for 'unknown'.

Reply:
I suppose if this is just used in mobile, might not be a huge deal? At first I thought this was shared.

Reply:
I think I made a commit here that will make it ok: https://github.com/leather-io/extension/pull/6177